### ISSUE TITLE
上传图片失败

### ISSUE THREAD

**Issue by chop2** [SwanHubX/self-hosted#40]
测试代码如下：
`
import swanlab as wandb
import numpy as np

wandb.init(
    project="test_swanlab",
    config={
        "learning_rate": 0.001,
        "batch_size": 32,
        "epochs": 10,
    }
)

img = np.random.randn(255,255,3)
img = img * 255.0
img = img.astype(np.uint8)

wandb.log({"test_img":wandb.Image(img,caption="test_image")})

wandb.log({"accuracy": 0.95, "loss": 0.05})
wandb.finish()
`

打印的log：
swanlab: Waiting for uploading complete
swanlab: Upload error: An error occurred (InvalidAccessKeyId) when calling the PutObject operation: The Access Key Id you provided does not exist in our records.

**Comment by Kaikaikaifang** [comment]
1. 检查是否为最新版本 v1.3
2. 若使用服务器部署，需哟啊检查服务器是否已经开放 9000 端口，图片等媒体资源需要使用 9000 端口

**Comment by chop2** [comment]
已经排查到问题。
似乎minio的服务只能绑定9000，9001端口。
但我的情况是这样，服务器上已经有一个另外的minio服务，我的问题如下：
    1.能否修改绑定的端口？(已尝试过，例如绑定9004，9005。但是Python sdk似乎只能连接9000端口)
    2.如果1不能实现，能否复用minio服务，但是docker compose如何修改呢？

期待回复

**Comment by Kaikaikaifang** [comment]
目前由于向 minio 发起请求时，端口会作为凭证校验的一部分，不支持修改 9000 端口
